src/main/java/duke/exceptions/DukeException.java

```diff
@@ -0,0 +1,7 @@
+package duke.exceptions;
+
+public class DukeException extends Exception {
+    public DukeException(String s) {
+        super(s);
+    }
+}
```

Comment:
Good use of creating a customized exception class for duke, but it may be better to make the name of the exception class more descriptive (e.g. EmptyDescriptiveException) and maybe create more exception classes which can extend from DukeException (instead of only by printing out different error message), which may make the type of exception more straightforward.

Reply:
Thanks for the suggestion! I'll add that in a future commit :)